examples/hkdf/src/hkdf.rs

```diff
@@ -53,8 +54,8 @@ fn check_output_limit(l: usize) -> Result<usize, HkdfError> {
 /// Output length l can be at most 255*HASH_LEN.
 pub fn expand(prk: &ByteSeq, info: &ByteSeq, l: usize) -> HkdfByteSeqResult {
     let n = check_output_limit(l)?;
-    let mut t_i = PRK::new();
-    let mut t = ByteSeq::new(n * PRK::capacity());
+    let mut t_i = PRK::new(); // PRK is of length HASH_SIZE
+    let mut t = ByteSeq::new(n * HASH_SIZE);
```

Comment:
Ugh, that's pretty ugly. But I guess we can live with it for now.

Reply:
Yeah this `<Array name>::capacity()` function basically breaks how we model arrays in the F* backend...